specs/lib/gltfPipelineSpec.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+var clone = require('clone');
```

Comment:
This reminds me that there are still other areas that use `Cesium.clone`. Those should probably switch over to use this library as well.